src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -64,11 +65,14 @@ export const post = (req: Request, res: Response) => {
 
 export const postSubmit = (req: Request, res: Response) => {
   logger.debugRequest(req, `${req.method} ${req.route.path}`);
-
-  const redirectUrl = isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS)
-    ? config.UPDATE_TRUSTS_SUBMISSION_INTERRUPT_URL
-    : config.UPDATE_CHECK_YOUR_ANSWERS_URL;
-
+  const appData: ApplicationData = getApplicationData(req.session);
+  const requiresTrusts: boolean = checkEntityRequiresTrusts(appData);
+  let redirectUrl = config.UPDATE_CHECK_YOUR_ANSWERS_URL;
+  if (requiresTrusts) {
```

Comment:
if (requiresTrusts && isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) {
  redirectUrl = getTrustLandingUrl(appData);
}
[instead of lines 71 to 75]

Reply:
Resolved